backend/endpoints/stocks/route.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+from ninja import Router, Schema
+from django.db import transaction
+from django.http.request import HttpRequest
+
+from lib.polygon_api import PolygonAPI, TickerDetails
+
+router = Router()
+
+
+class TickerDetailsSchema(Schema):
+    ticker: str
+    company_name: str
+    exchange: str
+
+
+@router.get("/all", response=List[TickerDetailsSchema])
+@transaction.atomic
+def all_stocks(request: HttpRequest) -> List[TickerDetails]:
```

Comment:
Try getting the supported stocks from the database - you might need to alter the schema to add company name and exchange. Could maybe add some logic that populates the name and exhange if the values are blank (so that we can easily add tickers in the admin interface without getting name + exchange exactly right).